src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 6;
+    private Random random = new Random();
+    private Figure figureObject;
+
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(FIGURE_COUNT);
+
+        switch (figureNumber) {
+            case 1 :
+                figureObject = new Circle();
```

Comment:
в конструктор фігур потрібно передавати рандомно згенеровані дані 
```suggestion
                figureObject = new Circle(randomRadius, colorSupplier.getRandomColor());
```